src/services/AudiusBackend.js

```diff
@@ -718,7 +731,10 @@ class AudiusBackend {
     } catch (err) {
       console.error(err)
     }
-    return feedItems
+    return feedItems.map(item => {
+      if (item.playlist_id) return AudiusBackend.getCollectionImages(item)
```

Comment:
nice -- so moving feed to v1 should help deal w/ this right?

Reply:
Yes, if we move feed and getPlaylists over the v1 then we shouldn't need this anymore!